sdk/python/kfp/_client.py

```diff
@@ -301,7 +301,7 @@ def list_pipelines(self, page_token='', page_size=10, sort_by=''):
     return self._pipelines_api.list_pipelines(page_token=page_token, page_size=page_size, sort_by=sort_by)
 
   # TODO: provide default namespace, similar to kubectl default namespaces.
-  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None, namespace=None):
+  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None, namespace=None, version_id=None):
```

Comment:
Could you add a comment on this to indicate that if pipeline id and pipeline version are specified at the same time, pipeline id will prevail and we recommend them to use pipeline version approach? Thanks! (BTW, later on, I'll change backend code to make pipeline version takes priority over pipeline id if both specified though)

Reply:
I added a comment about it but can you verify the wording to make sure it's accurate?